src/libs/Browser/types.ts

```diff
@@ -0,0 +1,13 @@
+type GetBrowser = () => string;
+
+type IsMobile = () => boolean;
+
+type IsMobileSafari = () => boolean;
+
+type IsMobileChrome = () => boolean;
+
+type IsSafari = () => boolean;
```

Comment:
I wonder if we need to have a few types that look the same. Maybe instead of that we can create a type like `type GetIsClientType = () => boolean` and use it instead of all of theese `Is*` types?
Also, if we type functions, the name should'n suggest that is a boolean like right now. Please avoid using `Is*` prefix in function typing.

Reply:
Still not sure what is the purpose of keeping the same types with different names.  

Even if we have similar parts in other places, we can be improved them in the future 😃 
Treat my comment as a nit, don't want to block Your PR

Reply:
As for naming, I would say that the type name should reflect what we are typing, so I would leave it as is. As for type reuse, it should be the choice of the team, I am not against any solution. From my point of view, in this case, there is no great advantage to creating a reusable type for `() => boolean`, because I don't see a case where we would want for example to change the type in every place we use it and I don't see much difference with `() => boolean` and `FnWhichReturnBoolean`. In other hand maybe it will look cleaner 😄